crates/bevy_time/src/common_conditions.rs

```diff
@@ -203,6 +203,38 @@ pub fn repeating_after_real_delay(
     }
 }
 
+/// Run condition that is active when the [`Time<Virtual>`] clock is paused.
+/// Use [`bevy_ecs::schedule::common_conditions::not`] to make it active when
+/// it's not paused.
+///
+/// ```rust,no_run
+/// # use bevy_app::{App, NoopPluginGroup as DefaultPlugins, PluginGroup, Update};
+/// # use bevy_ecs::schedule::{common_conditions::not, IntoSystemConfigs};
+/// # use bevy_time::common_conditions::paused;
+/// fn main() {
+///     App::new()
+///         .add_plugins(DefaultPlugins)
+///         .add_systems(
+///             Update,
+///             (
+///                 is_paused.run_if(paused()),
+///                 not_paused.run_if(not(paused())),
+///             )
+///         )
+///     .run();
+/// }
+/// fn is_paused() {
+///     // ran when time is paused
+/// }
+///
+/// fn not_paused() {
+///     // ran when time is not paused
+/// }
+/// ```
+pub fn paused() -> impl FnMut(Res<Time<Virtual>>) -> bool + Clone {
+    move |time: Res<Time<Virtual>>| time.is_paused()
+}
```

Comment:
why not just make this like the following?
```suggestion
pub fn paused(time: Res<Time<Virtual>>) -> bool {
    time.is_paused()
}
```

the extra parenthesis seem to add extra boilerplate.

Reply:
I agree

Reply:
That won't compile. This isn't a regular system but a run condition. It needs the closure.

Reply:
Run conditions are just systems. In most of the other cases, we needed to generate a `Local`, necessitating the dance with closures.

Reply:
Ah yep, I forgot to remove the "()" from the test when I tried that :(